create_tables.py

```diff
@@ -1,29 +1,38 @@
+# -- coding: utf-8 --
+"""This script create the Sparkify database and all its tables. """
 import psycopg2
 from sql_queries import create_table_queries, drop_table_queries
+import argparse
 
 
-def create_database():
+def create_database(user_name: str, password: str):
```

Comment:
Thanks for annotating this function. Could you also add the return type hints?

Reply:
Done

Reply:
Thansk!